src/Microsoft.AspNet.Localization/ApplicationBuilderExtensions.cs

```diff
@@ -16,17 +17,26 @@ public static class ApplicationBuilderExtensions
         /// requests based on information provided by the client using the default options.
         /// </summary>
         /// <param name="builder">The <see cref="IApplicationBuilder"/>.</param>
+        /// <param name="defaultRequestCulture">The default <see cref="RequestCulture"/> to use if none of the
+        /// requested cultures match supported cultures.</param>
         /// <returns>The <see cref="IApplicationBuilder"/>.</returns>
-        public static IApplicationBuilder UseRequestLocalization(this IApplicationBuilder builder)
+        public static IApplicationBuilder UseRequestLocalization(
+            this IApplicationBuilder builder,
```

Comment:
BTW since you're here, I think @DamianEdwards prefers that the IAppBuilder parameter type is always named `app`.